content/03.results.md

```diff
@@ -119,32 +139,44 @@ Colors and shapes represent the 12 strains and two clades, respectively.
+The *VCF graph* only exhibited higher mappability on the reference strain *S.c. S288C*, and the difference was marginal.
+We also found that the genetic distance to the reference strain correlates with the better mapping on the *cactus graph* over the *VCF graph* (Pearson's product-moment correlation, p-value=1.415e-11).
 Consequently, the benefit of using the *cactus graph* is largest for strains in the *S. paradoxus* clade and smaller for reads from strains in the *S. cerevisiae* clade.
-We observed a similar trend when exploring the mapping identity of the short reads on the graphs (see Figure {@fig:3}b and [Supplementary Information](#supplementary-information)).
+These results suggest that the improvement in mappability is not driven by the higher sequence content in the *cactus graph* alone (15.4 Mb compared to 12.4 Mb in the *VCF graph*).
+We observed similar trends in the mapping identity of the short reads as with mapping quality (see Figure {@fig:3}b and [Supplementary Information](#supplementary-information)).
+
+Interestingly, our measurements did not show a substantial difference between strains that were used to construct the graph and the other strains. 
+Only the number of alignments with perfect identity is substantially lower for the strains that were not included in the creation of the graphs (see Figure {@fig:3}b). 
+This suggests that two strains from each clade as well as the reference strain are sufficient to capture most of the genetic variation among all the strains. 
+For a direct comparison, see Figure {@fig:panel5} which shows results of the same experiment on graphs generated from all 12 strains.
 
-Interestingly, our measurements did not show a substantial difference between strains included in the graph and excluded strains. The results suggest that two strains from each clade as well as the reference strain are sufficient to capture most of the genetic variation among all the strains. Only the number of alignments with perfect identity is substantially lower for the strains that were not included in the creation of the graphs (see Figure {@fig:3}b). For a direct comparison, see Figure {@fig:panel5} which shows results of the same experiment on graphs generated from all 12 strains.
+[//]: # (The figure mentioned in the previous sentence doesn't seem to be included here.)
 
-Next, we compared the SV genotype performance of both graphs.
+[//]: # (Here too I think identity is better than MAPQ as a metric. With MAPQ you have confound of repetitiveness, where correctly adding repetitive sequence could end up lowering your MAPQ and making you look worse. What's the reasoning for including both MAPQ and identity? If we have a strong reason for it, we should make it explicit in the text because it's not clear to me.)
```

Comment:
Yeast isn't exactly known for repetitive sequence, but the point about explaining the measures is a good one.